iol-core/fhir_to_hl7_mapper.bal

```diff
@@ -0,0 +1,53 @@
+import ballerinax/health.fhir.r4;
+import ballerinax/health.fhir.r4.international401;
+
+// Function to map a FHIR Patient resource to HL7v2
+public isolated function mapFhirPatientToHL7(
```

Comment:
Let's use datamapper. input will be FHIR patient record and output will be RSP_K21